tests/test_merge.py

```diff
@@ -43,6 +44,26 @@ def take_first(conflict, _, __, ___):
         self.assertEqual(m.unified_patches,
                          [('change', 'changeme', ('Jo', 'Joe'))])
 
+    def test_continue_run_multiple_conflicts_per_patch(self):
+        lca = {'foo': [{'x': 1}, {'y': 2}]}
+        first = {'foo': [{'x': 1}, {'y': 2}, {'z': 4}]}
+        second = {'bar': 'baz'}
+
+        expected = {
```

Comment:
This tests inherits from `unittest.TestCase` so that's why `parametrize` does not work ([SO](http://stackoverflow.com/questions/18182251/does-pytest-parametrized-test-work-with-unittest-class-based-tests)).
I'll just change to `items()` iteration and I'll leave it as is for the time being.

The follow-up solutions for this would either be:
* Require nose-parametrize for parametrizing `unitest.TestCase` test methods
* Port everything to `pytest`